CMakeLists.txt

```diff
@@ -247,7 +247,6 @@ if(BUILD_CLIENT)
     if(NOT BUILD_LIBRARIES_ONLY)
         add_subdirectory(doc)
         add_subdirectory(doc/dev)
-        add_subdirectory(admin)
```

Comment:
Was this really in the diff? Looks like an accident or something - it looks pretty wrong, don't even know what it would be good for in the linux packages